torchgeo/datasets/landcoverai.py

```diff
@@ -50,7 +52,223 @@ class LandCoverAI(NonGeoDataset):
+        """
+        if self._verify_data():
+            return
+
+        # Check if the zip file has already been downloaded
+        pathname = os.path.join(self.root, self.filename)
+        if os.path.exists(pathname):
+            self._extract()
+            return
+
+        # Check if the user requested to download the dataset
+        if not self.download:
+            raise RuntimeError(
+                f"Dataset not found in `root={self.root}` and `download=False`, "
+                "either specify a different `root` directory or use `download=True` "
+                "to automatically download the dataset."
+            )
+
+        # Download the dataset
+        self._download()
+        self._extract()
+
+    @abc.abstractmethod
+    def _verify_data(self) -> bool:
+        """Verify if the images and masks are present."""
+
+    def _download(self) -> None:
+        """Download the dataset."""
+        download_url(self.url, self.root, md5=self.md5 if self.checksum else None)
+
+    def _extract(self) -> None:
+        """Extract the dataset."""
+        extract_archive(os.path.join(self.root, self.filename))
+
+    def plot(
+        self,
+        sample: Dict[str, Tensor],
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> plt.Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample returned by :meth:`__getitem__`
+            show_titles: flag indicating whether to show titles above each panel
+            suptitle: optional string to use as a suptitle
+
+        Returns:
+            a matplotlib Figure with the rendered sample
+
+        .. versionadded:: 0.2
+        """
+        image = np.rollaxis(sample["image"].numpy().astype("uint8").squeeze(), 0, 3)
+        mask = sample["mask"].numpy().astype("uint8").squeeze()
+
+        num_panels = 2
+        showing_predictions = "prediction" in sample
+        if showing_predictions:
+            predictions = sample["prediction"].numpy()
+            num_panels += 1
+
+        fig, axs = plt.subplots(1, num_panels, figsize=(num_panels * 4, 5))
+        axs[0].imshow(image)
+        axs[0].axis("off")
+        axs[1].imshow(mask, vmin=0, vmax=4, cmap=self._lc_cmap, interpolation="none")
+        axs[1].axis("off")
+        if show_titles:
+            axs[0].set_title("Image")
+            axs[1].set_title("Mask")
+
+        if showing_predictions:
+            axs[2].imshow(
+                predictions, vmin=0, vmax=4, cmap=self._lc_cmap, interpolation="none"
+            )
+            axs[2].axis("off")
+            if show_titles:
+                axs[2].set_title("Predictions")
+
+        if suptitle is not None:
+            plt.suptitle(suptitle)
+        return fig
+
+
+class LandCoverAIGeo(LandCoverAI, RasterDataset):
+    r"""LandCover.ai Geo dataset.
+
+    See the abstract base LandCoverAI class to find out more.
+    """
+    filename_glob = "images/*.tif"
```

Comment:
```suggestion
    filename_glob = os.path.join("images", "*.tif")
```
This will work better on Windows.

I've never tried using a path in `filename_glob` before but I think it should work.